src/organisms/cards/AddCard/index.js

```diff
@@ -1,14 +1,42 @@
 import React from 'react';
 import styles from './add_card.module.scss';
 
-function AddCard ( { children } ) {
+function renderContent( props ) {
+  const {
+    children,
+    beneficiary,
+    jewelry,
+  } = props;
+
+  var txt, icon;
+
+  if ( beneficiary ) {
+    txt = 'Add Beneficiary';
+    icon = 'plus';
+  }
+
+  if ( jewelry ) {
+    txt = 'Add jewelry';
+    icon = 'jewelry';
+  }
+
   return (
-    <div className={ styles.add }>{ children }</div>
+    <div>
+      <Icon
+        icon={ icon }
+        className={ styles.icon }
+      />
+      { txt }
+    </div>
   )
 }
 
-AddCard.defaultProps = {
-  children: '+ Add New Beneficiary'
+function AddCard ( props ) {
+  return (
+    <div className={ styles.add }>
+      { renderContent(props) }
```

Comment:
would it make more sense to accept the icon name and text as props? otherwise every time we add a new `AddCard` type, we'll have to modify this component